src/app/download/download.component.ts

```diff
@@ -16,22 +16,22 @@ export class DownloadComponent {
     zip.file('AmpAd.html', data);
```

Comment:
By convention, this should be named 'index.html'. Typically ad primary files are named this way. If you export an ad bundle zip from https://ampstoryads.web.app/ you'll notice the same convention used in their zip.